app.rb

```diff
@@ -0,0 +1,138 @@
+require_relative 'book'
+require_relative 'classroom'
+require_relative 'person'
+require_relative 'rental'
+require_relative 'student'
+require_relative 'teacher'
+
+class App
+  attr_accessor :books, :people, :rentals
+
+  def initialize
+    @books = []
+    @rentals = []
+    @people = []
+  end
+
+  def list_books
+    if @books.empty?
+      puts 'Currently, there are no books!'
+    else
+      @books.each_with_index do |book, i|
+        puts "#{i}) Title: #{book.title}, Author: #{book.author}"
+      end
+    end
+  end
+
+  def list_people
```

Comment:
- [ ] Kindly fix this problem in the `list all people` option for displaying the `Teacher` as for teacher age display the name and for teacher name display age.
![image](https://github.com/Arch-Noize/oop-library/assets/24703171/6c099703-8f5f-40ee-9465-c45bc0894175)